src/Core/src/Core/IBorder.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Microsoft.Maui
+{
+	/// <summary>
+	/// Provides functionality to set the view borders.
+	/// </summary>
+	public interface IBorder
+	{
+		/// <summary>
+		/// Gets the the color of a view's border.
+		/// </summary>
+		Color BorderColor { get; }
+
+		/// <summary>
+		/// Gets the the width of a view's border.
+		/// </summary>
+		double BorderWidth { get; }
```

Comment:
Maybe it could be `Thickness` type to not apply the same border width on each side? (WinUI [Border.BorderThickness](https://docs.microsoft.com/en-us/windows/winui/api/microsoft.ui.xaml.controls.border.borderthickness))